src/proposals/proposal.cpp

```diff
@@ -233,6 +303,7 @@ namespace hypha
 
     std::pair<bool, checksum256> Proposal::hasOpenProposal(name proposalType, checksum256 docHash)
     {
+      TRACE_FUNCTION()
```

Comment:
What is this for and does this impact mainnet performance?

Reply:
It's for tracing. It logs the file and line.
I'm unsure, but maybe. It can be disabled IIRC. @Gerard097  might know more.